src/libs/actions/Report.ts

```diff
@@ -755,7 +741,7 @@ function openReport(
     const optimisticReport = reportActionsExist(reportID)
         ? {}
         : {
-              reportName: allReports?.[reportID]?.reportName ?? CONST.REPORT.DEFAULT_REPORT_NAME,
+              reportName: ReportConnection.getAllReports()?.[reportID]?.reportName ?? CONST.REPORT.DEFAULT_REPORT_NAME,
```

Comment:
Should this not be more like this?
```suggestion
              reportName: ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`]?.reportName ?? CONST.REPORT.DEFAULT_REPORT_NAME,
```